src/NuGet.Core/NuGet.Commands/PublicAPI/netstandard2.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,1197 @@
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoWrapper.DirectoryInfoWrapper(System.IO.DirectoryInfo directoryInfo, bool isParentPath = false) -> void
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase.FileInfoBase() -> void
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoWrapper
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoWrapper.FileInfoWrapper(System.IO.FileInfo fileInfo) -> void
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileSystemInfoBase
+Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileSystemInfoBase.FileSystemInfoBase() -> void
+Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch
+Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Equals(Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch other) -> bool
+Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.FilePatternMatch(string path, string stem) -> void
+Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Path.get -> string
+Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Stem.get -> string
+Microsoft.Extensions.FileSystemGlobbing.Internal.ILinearPattern
+Microsoft.Extensions.FileSystemGlobbing.Internal.ILinearPattern.Segments.get -> System.Collections.Generic.IList<Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment>
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment.CanProduceStem.get -> bool
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment.Match(string value) -> bool
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPattern
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPattern.CreatePatternContextForExclude() -> Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPattern.CreatePatternContextForInclude() -> Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext.Declare(System.Action<Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment, bool> onDeclare) -> void
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext.PopDirectory() -> void
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext.PushDirectory(Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoBase directory) -> void
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext.Test(Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoBase directory) -> bool
+Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext.Test(Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase file) -> Microsoft.Extensions.FileSystemGlobbing.Internal.PatternTestResult
+Microsoft.Extensions.FileSystemGlobbing.Internal.IRaggedPattern
```

Comment:
Ideally yes, but we already have been, so it's out of scope for this PR which is simply starting to use the analyzers to tell us about these types of issues.

Reply:
Those are brought in through submodules.
See https://github.com/NuGet/Home/issues/2345. This is one of those exceptional cases where if we remove these public APIs, I wouldn't feel bad :) In fact others want us to do it. 

We should revisit https://github.com/NuGet/NuGet.Client/pull/2675/files at some point. Ensure everything is source-buildable first, since that was the blocker last time.